rinja_parser/src/expr.rs

```diff
@@ -192,9 +209,9 @@ impl<'a> Expr<'a> {
     expr_prec_layer!(or, and, "||");
     expr_prec_layer!(and, compare, "&&");
     expr_prec_layer!(compare, bor, "==", "!=", ">=", ">", "<=", "<");
-    expr_prec_layer!(bor, bxor, "|");
-    expr_prec_layer!(bxor, band, "^");
-    expr_prec_layer!(band, shifts, "&");
+    expr_prec_layer_keyword!(bor, bxor, "bitor", "|");
```

Comment:
Thinking out loud for later: would be nice to still parse `^` and `&` to mention to use the full text alternative. For `|`, need to think about it (where to make it checked more exactly).

Reply:
Yes, I re-added them.